docker/ProdDjangoDockerfile

```diff
@@ -4,39 +4,41 @@ FROM node:${NODE_VER} AS node-assets
 # Make npm output less verbose
 ENV NPM_CONFIG_LOGLEVEL warn
 
-# Workaround to avoid webpack hanging, see:
-# https://github.com/webpack/webpack-dev-server/issues/128
-ENV UV_THREADPOOL_SIZE 128
-
 # Oddly, node-sass requires both python and make to build bindings
 RUN apk add --no-cache paxctl python make g++
 RUN paxctl -cm /usr/local/bin/node
 
-COPY ./ /src-files
-RUN cd /src-files && ( npm install && npm run build )
+# Install gulp globally so it can be used during builds
+RUN NPM_CONFIG_UNSAFE_PERM=true npm install --global gulp-cli
```

Comment:
The re-inclusion of `gulp` is a regression: remove the install task for gulp, and reinstate the `npm install && npm run build` task above. See here for reference: https://github.com/freedomofpress/securethenews/pull/181/files#diff-39520f27c7fd6ecc9b11465fde9fff66R12